Extensions/HomePageExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using Infrastructure;
+using Infrastructure.Pages;
+using Core;
+
+namespace Extensions
+{
```

Comment:
For all the functions here, I would change the name of the "place" parameter to perhaps productIndex?